onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -803,10 +805,11 @@ def aten_clamp_min(self, min_):
     return result
 
 
-def aten_clone(self: TensorType, memory_format: Optional[str] = None) -> TensorType:
+@torch_op("aten::clone")
+def aten_clone(self: TensorType, memory_format: str = None) -> TensorType:
     # clone(Tensor self, *, MemoryFormat? memory_format=None) -> Tensor
 
-    raise NotImplementedError()
+    return op.CastLike(self, self)
```

Comment:
Can this be Identity? (e.g. return self) IIUC onnx tensors are immutable.

Reply:
good idea